packages/app-desktop/gui/note-viewer/index.html

```diff
@@ -589,6 +589,22 @@
 			let element = event.target;
 
 			// To handle right clicks on resource icons
+			let mermaidElement = element.closest("[id^='mermaid']") || ((element.firstChild && element.firstChild.id?.startsWith('mermaid')) ? element.firstChild : null);
```

Comment:
Please use a better test than that because other elements could start with mermaid. Isn't a class name set on the element? If not, could you set one?